templates/security/cves/about.html

```diff
@@ -0,0 +1,438 @@
+                  <i class="p-icon--help"></i>Needs evaluation
+                </td>
+                <td colspan="2">The vulnerability of this package is not known for the given release. It needs to be evaluated.</td>
+              </tr>
+              <tr>
+                <td class="p-table__cell--icon-placeholder">
+                  <i class="p-icon--success"></i>Not affected
+                </td>
+                <td colspan="2">
+                  The package, while related to the CVE in some way, is not affected by the issue for the given release. We usually provide an explanation, either next to the status or in the notes.
+                </td>
+              </tr>
+              <tr>
+                <td class="p-table__cell--icon-placeholder">
+                  <i class="p-icon--warning"></i>Vulnera&shy;ble
+                </td>
+                <td colspan="2">The package is affected by the issue for the given release.</td>
+              </tr>
+              <tr>
+                <td class="p-table__cell--icon-placeholder">
+                  <i class="p-icon--warning"></i>Vulnera&shy;ble, work in progress
+                </td>
+                <td colspan="2">
+                  The package is affected by the issue for the given release, and the Ubuntu security team is currently working on a fix.
+                </td>
+              </tr>
+              <tr>
+                <td class="p-table__cell--icon-placeholder">
+                  <i class="p-icon--warning"></i>Vulnera&shy;ble, fix deferred
+                </td>
+                <td colspan="2">
+                  The package is affected by the issue for the given release, but fixing has been deferred. We usually provide an explanation, either next to the status or in the notes.
+                </td>
+              </tr>
+              <tr>
+                <td class="p-table__cell--icon-placeholder">
+                  <i class="p-icon--success"></i>Fixed
+                </td>
+                <td colspan="2">
+                  The package was affected by the issue for the given release, but a version fixed by the Ubuntu Security team is now available. We usually indicate the package version that contains the patch.
+                </td>
+              </tr>
+              <tr>
+                <td class="p-table__cell--icon-placeholder">
+                  <i class="p-icon--error-grey"></i>Ignored
+                </td>
+                <td colspan="2">
+                  The package, while related to the CVE in some way for the given release, won’t be fixed. This could be because the package is no longer supported, the CVE has been withdrawn,
+                  or fixing the issue would break other functionality. We usually provide an explanation, either next to the status or in the notes.
+                </td>
+              </tr>
+              <tr>
+                <td class="p-table__cell--icon-placeholder">Not in release</td>
```

Comment:
On the design this one and "Not affected" have no icon and are aligned on the grid like so 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/f87f5cd2-501b-462e-97e2-c7e3b878f187)

Reply:
I'm currently following the icons used in the copy docs since it is also similar to the icons in the [specs](https://docs.google.com/document/d/1Q0wPWR7ty1ewZiaRLsX6T2kGCp_Kaag8_FAFqH-cU1g/edit#heading=h.lp2utri126o4). They are also the status icons that I'm using in the CVEs detailed search view. 
Should it be changed to follow the icons in the design instead?